OpenSim/Common/ModelDisplayHints.h

```diff
@@ -53,6 +52,7 @@ your component produces. The currently-supported flags are:
   - show frames
   - show labels
   - show debug geometry
+  - display_scale_factor
```

Comment:
The other items in this list do not have underscores. I think the list should either all use the exact property names, or all use the more casual phrase.